libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/roadobject/distanceinfo/RoadObjectDistanceInfoType.kt

```diff
@@ -0,0 +1,54 @@
+package com.mapbox.navigation.base.trip.model.roadobject.distanceinfo
+
+import androidx.annotation.IntDef
+
+/**
+ * Holds available [RoadObjectDistanceInfo] types.
+ *
+ * Available values are:
+ * - [RoadObjectDistanceInfoType.GANTRY]
+ * - [RoadObjectDistanceInfoType.LINE]
+ * - [RoadObjectDistanceInfoType.POINT]
+ * - [RoadObjectDistanceInfoType.POLYGON]
+ * - [RoadObjectDistanceInfoType.SUB_GRAPH]
+ */
+object RoadObjectDistanceInfoType {
+
+    /**
+     * Type of the [GANTRY].
+     */
+    const val GANTRY = 0
+
+    /**
+     * Type of the [LINE].
+     */
+    const val LINE = 1
+
+    /**
+     * Type of the [POINT].
+     */
+    const val POINT = 2
+
+    /**
+     * Type of the [POLYGON].
+     */
+    const val POLYGON = 3
+
+    /**
+     * Type of the [SUB_GRAPH].
```

Comment:
What is a sub graph? Where do we best document this?

Reply:
cc @etl @SiarheiFedartsou

Reply:
We are going to add a new type of objects, subgraph. It's appropriate for tunnels which have unusual form, with two entrances and three exits, for instance (e.g. carparks)